src/mlpack/methods/preprocess/preprocess_imputer_main.cpp

```diff
@@ -68,16 +69,16 @@ using namespace data;
 
 void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
 {
-  const string inputFile = params.Get<string>("input_file");
-  const string outputFile = params.Get<string>("output_file");
+  arma::mat input = params.Get<arma::mat>("input");
```

Comment:
```suggestion
  arma::mat input = std::move(params.Get<arma::mat>("input"));
```

if we take a look at the output:

```
[WARN ] --dimension is not specified; the imputation will be applied to all dimensions.
[WARN ] The file does not contain any user-defined missing variables. The program did not perform and
```

we can see `output` is never set since we can't find any of the `nans`.